.vscode/settings.json

```diff
@@ -10,4 +10,9 @@
     },
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
     "typescript.tsc.autoDetect": "off",
+    "yaml.schemas": {
```

Comment:
No need to commit this here...

Reply:
I specifically wanted to include this, to get the local project looking live at the local schema, as its changed